contributing/getting-started-e2e.md

```diff
@@ -173,6 +173,7 @@ Now that you have all of the files copied locally you can actually build the cod
   An alternative to installing `yarn` is to invoke each Yarn command in this guide with `npx yarn` during local development. This will automatically use the current stable version of `yarn`.
 
 * Closure Compiler is automatically installed by Yarn, but it requires Java 8 which you need to install separately. AMP's version of Closure Compiler won't run with newer versions of Java. Download an installer for Mac, Linux or Windows [here](http://www.oracle.com/technetwork/java/javase/downloads/jre8-downloads-2133155.html).
+  * Note: If you have several versions of Java installed, make sure to be using Java 8 by updating the environment variable in your bash profile: ```export JAVA_HOME=`/usr/libexec/java_home -v 1.8```.
```

Comment:
The triple backticks should be in separate lines for the code be shown as a callout like this:
```
export JAVA_HOME=`/usr/libexec/java_home -v 1.8
```
So you'll need to write this as follows (minus the spaces):

```
` ` `
export JAVA_HOME=`/usr/libexec/java_home -v 1.8
` ` `

```

Reply:
Also restate the text to:

```
If you have multiple versions of Java installed, make sure you are using Java 8 by adding this to `~/.bashrc`:
````